### DISCUSSION TITLE
\fdusetup里面设置了biblatex，如何让后面的参考文献不显示url, doi那些信息？

### DISCUSSION THREAD

**Discussion by iamforerunner** [stone-zeng/fduthesis#300] [Q&A]
\fdusetup里面有关biblatex的选项较少

在后面重新 \usepackage[...]{biblatex}
在方括号中...处填入大量禁止url, doi那些选项则会报错

这要怎么解决啊？

**Comment by note286** [comment]
移除`.bib`中的相应行。

**Reply by iamforerunner** [reply]
> 移除`.bib`中的相应行。

% \usepackage[backend=biber,
%   style=gb7714-2015ay,
%   gbalign=right,
%   gbpub=false,
%   gbnoauthor=false,
%   gbbiblabel=bracket,
%   gbnamefmt=uppercase,
%   gbtype=true,
%   gbmedium=false,
%   gbfieldtype = false,
%   gbpunctin=false,
%   gbtitlelink=false,
%   gblabelref=true,
%   gbcitelocal=gb7714-2015,  % 正文引用的时候
%   gbbiblocal=gb7714-2015, % 最后参考文献的时候 
%   mergedate=none,  % 参考文献的日期
%   gblanorder=chineseahead,
%   gbannote=false,
%   gbctexset=true,
%   gbcodegbk=false,
%   gbstyle=true,
%   gbfootbib=false,
%   gbfnperpage=false,
%   gbstrict=true,
%   gbfieldstd=false,
%   url=false,
%   giveninits =true,
%   uniquelist =minyear,
%   maxbibnames =3,
%   minbibnames =3,
%   maxcitenames =2,
%   mincitenames =1,
%   hyperref =auto,
%   backref =false,
%   refsection =none,
%   refsegment =none,
%   citereset =none,
%   labeldate =year,
%   doi=false,
%   isbn=false,
%   eprint=false,
%   % sorting=multipinyin,
%   sortlocale=zh__pinyin]{biblatex}
% \addbibresource[location=local]{./ref.bib} % 参考文献位置

上一个版本里，单独设置biblatex不会报错，这个版本好像就不行了……

**Comment by stone-zeng** [comment]
在 `\fdusetup` 前使用 `\PassOptionsToPackage{...}{biblatex}` 添加选项。
